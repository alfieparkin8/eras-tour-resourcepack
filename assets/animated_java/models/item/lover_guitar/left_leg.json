{
	"textures": {
		"0": "animated_java:item/lover_guitar/blank_hair"
	},
	"elements": [
		{
			"from": [
				8,
				7.5,
				7
			],
			"to": [
				12,
				19.5,
				11
			],
			"rotation": {
				"angle": 0,
				"axis": "y",
				"origin": [
					0,
					8,
					1
				]
			},
			"faces": {
				"north": {
					"uv": [
						1,
						5,
						2,
						8
					],
					"texture": "#0"
				},
				"east": {
					"uv": [
						0,
						5,
						1,
						8
					],
					"texture": "#0"
				},
				"south": {
					"uv": [
						3,
						5,
						4,
						8
					],
					"texture": "#0"
				},
				"west": {
					"uv": [
						2,
						5,
						3,
						8
					],
					"texture": "#0"
				},
				"up": {
					"uv": [
						2,
						5,
						1,
						4
					],
					"texture": "#0"
				},
				"down": {
					"uv": [
						3,
						4,
						2,
						5
					],
					"texture": "#0"
				}
			}
		}
	],
	"display": {
		"head": {
			"rotation": [
				0,
				180,
				0
			]
		}
	}
}