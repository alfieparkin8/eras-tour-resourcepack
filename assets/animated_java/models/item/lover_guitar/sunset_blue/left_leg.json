{"parent":"animated_java:item/lover_guitar/left_leg","textures":{"1":"minecraft:block/guitars/lover_blue"}}