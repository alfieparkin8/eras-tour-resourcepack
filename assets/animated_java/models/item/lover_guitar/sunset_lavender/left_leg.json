{
	"parent": "animated_java:item/lover_guitar/left_leg",
	"textures": {
		"2": "animated_java:item/lover_guitar/lover_lavender"
	}
}