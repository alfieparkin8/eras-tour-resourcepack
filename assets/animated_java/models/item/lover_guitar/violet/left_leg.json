{
	"parent": "animated_java:item/lover_guitar/left_leg",
	"textures": {
		"1": "animated_java:item/lover_guitar/violet_layer_1",
		"5": "animated_java:item/lover_guitar/violet_boots_layer_1"
	}
}