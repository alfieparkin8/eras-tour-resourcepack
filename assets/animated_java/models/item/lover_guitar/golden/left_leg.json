{"parent":"animated_java:item/lover_guitar/left_leg","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1"}}